flixkit.go

```diff
@@ -114,3 +134,128 @@ func FetchFlixWithContext(ctx context.Context, url string) (string, error) {
 
 	return string(body), nil
 }
+
+func (s *flixServiceImpl) GetAndReplaceCadenceImports(ctx context.Context, templateName string, network string) (*FlowInteractionTemplateExecution, error) {
+	template, err := s.GetTemplate(ctx, templateName)
+	if err != nil {
+		return nil, err
+	}
+	var execution FlowInteractionTemplateExecution
+	var cadenceCode string
+	var replaceableCadence FlowInteractionTemplateCadence
+	if replaceableCadence, err = v1_1.ParseFlix(template); err == nil {
```

Comment:
i think we talked about this one to not do this but to use switch case instead. Just noting it here in case we forgot